src/components/ec-menu/ec-menu.vue

```diff
@@ -80,12 +80,13 @@ function getLinkDataTest(link) {
 
   &__item {
```

Comment:
why don't we use here truncate tailwind class?

Reply:
Not sure if this would ok, items can also be too elements on the same line(horizontal). Not sure if applying trancate to those would play well.